CMGTools/MonoXAnalysis/python/tools/eventVars_monojet.py

```diff
@@ -3,8 +3,8 @@
 from CMGTools.MonoXAnalysis.tools.BTagWeightCalculator import BTagWeightCalculator
```

Comment:
@emanueledimarco 
Why the BTagWeightCalculator is inside the MonoXAnalysis ? IS there something specific, there are other duplicate in other packages.